spec/requests/api/eform_spec.rb

```diff
@@ -114,7 +114,7 @@
 
       it 'retrieves TA proxy object when authenticating via token with TA key' do
         expect(UpvsEnvironment).to receive(:upvs_proxy).with(assertion: nil).and_call_original.at_least(:once)
-        expect(UpvsProxy).to receive(:new).with(hash_not_including('upvs.sts.saml.assertion')).and_call_original.once
+        expect(UpvsProxy).to receive(:new).with(hash_not_including('upvs.sts.saml.assertion')).and_return(double).once
```

Comment:
Nie je lepsie aby to volalo naozaj to pod tym? Je to potom viac e2e test

Reply:
Ano, to aj bol zamer, no volanie `UpvsProxy.new` si ziada ENV, ktore nemame nastavene na CI... spravim na to issue?

Reply:
Aha rozumiem, potom nechajme takto, asi nechceme nech to vola upvs z testov.

Reply:
Ono to UPVS realne nezavola lebo https://github.com/slovensko-digital/slovensko-sk-api/pull/43/files#diff-8769e86d3e53028f8f6a541a1e16eadaR89 len to realne vytvori proxy objekt. Ale nechajme takto, je to ok podla mna.